test/acceptance-tests/operator_images_test.go

```diff
@@ -0,0 +1,66 @@
+package acceptance_tests
+
+import (
+	"encoding/json"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/securesign/structural-tests/test/support"
+	"log"
+)
+
+var _ = Describe("Trusted Artifact Signer Operator", Ordered, func() {
+
+	var (
+		snapshotImages support.SnapshotMap
+		operatorImages support.OperatorMap
+		operator       string
+	)
+
+	It("get and parse snapshot.json file", func() {
```

Comment:
It would be very common use-case. I would create function for getting the SnapshotMap

Reply:
Done